hyperdash/client.py

```diff
@@ -1,3 +1,17 @@
+from .sdk_message import create_metric_message
+import numbers
+import types
+
 class HDClient:
-    def __init__(self, logger):
+    def __init__(self, logger, server_manager, sdk_run_uuid):
         self.logger = logger
+        self.server_manager = server_manager
+        self.sdk_run_uuid = sdk_run_uuid
+
+    def metric(self, value, name):
+        assert isinstance(value, numbers.Real), 'value must be a real number.'
+        assert isinstance(name, types.StringType)
```

Comment:
is types.StringType what we want here? I usually import the StringType from six since it changes from python version to python version

Reply:
This function should probably also print something right?